### DISCUSSION TITLE
Approved student account lost access to Copilot today

### DISCUSSION THREAD

**Discussion by h4nkyn** [community/community#60962] [Copilot Conversations] [answered]
### Topic

Bug/Account Error

### Body

 I have had Copilot for about around a year now as a student but I just lost it randomly today. Github still recognizes me as a verified student with the student developer pack. I received the following error in my output for Copilot in VSCode:

[INFO] [auth] [2023-07-16T02:20:00.938Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-07-16T02:20:00.938Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as h4nkyn."

I attempted logging in and out of Copilot in VSCode with no luck. Same error. When I visit my settings page under Copilot, it only offers me to purchase it. If anyone can help me diagnose this it would be greatly appreciated. Thank you!

**Comment by davecheney** [comment, marked as answer]
@h4nkyn your student coupon has expired. If you are still an active student you may re-apply for the Student Developer Pack, https://education.github.com/pack
